crates/xtask-unpublished/src/xtask.rs

```diff
@@ -226,6 +246,22 @@ fn output_table(results: Vec<Record>) -> fmt::Result {
     Ok(())
 }
 
+fn output_version_bump_notice(results: &[Record]) {
+    let pkgs_need_bump = results
+        .iter()
+        .filter_map(|(name, .., published)| published.then_some(name.clone()))
+        .collect::<Vec<_>>();
+
+    if !pkgs_need_bump.is_empty() {
+        print!("### :warning: ");
+        println!("Require at least a patch version bump for each of the following packages:\n");
+        for pkg in pkgs_need_bump {
+            println!("* {pkg}");
+        }
+        println!()
+    }
+}
```

Comment:
On its own, this doesn't make sense.  This is reporting unpublished crates as needing a bump.  This only makes sense based on how this script is being called (the later commits).